### PR TITLE
Make Jitsi download script a JS script

### PR DESCRIPTION
**Pull request by turt2live**
It's awful but means that we create the directory on all platforms in-process.

Fixes https://github.com/vector-im/riot-web/issues/13215

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7645,8 +7645,8 @@ matrix-mock-request@^1.2.3:
     create-react-class "^15.6.0"
     diff-dom "^4.1.3"
     diff-match-patch "^1.0.4"
-    emojibase-data "^5.0.1"
-    emojibase-regex "^4.0.1"
+    emojibase-data "^4.0.2"
```
Comment (jryans) [inline yarn.lock:7648]: This seems wrong / unrelated...?

File: scripts/build-jitsi.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// This is a JS script so that the directory is created in-process on Windows.
```
Comment (jryans) [inline scripts/build-jitsi.js:1]: Can you expand this to elaborate why it's need on Windows? Did the shell script version allow mkdir to complete before the directory actually exists...?

Comment (turt2live) [reply scripts/build-jitsi.js:1]: There's a risk of that, though the more concerning problem is that unless the Windows user has set up all their file associations correctly (unlike me, apparently) the script just won't run.

File: yarn.lock (hunk 1 above)
Comment (turt2live) [reply yarn.lock:7648]: yarn refused to install the new dependency without doing this :(

**Pull request merged**